source/tools/testConsistency/dotnetWPF-renderer/WPFRenderer/WPFRendererTests/UnitTest1.cs

```diff
@@ -0,0 +1,137 @@
+using System;
```

Comment:
this file should probably have a more descriptive name (as should the test class)